src/components/hooks/useFetch.ts

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+
+import { fetchDataService } from '../../services/getResponseDataService';
+
+export interface IArticles {
```

Comment:
shouldn't this be moved to another file? I don't see it's used here

Reply:
and why plural? maybe just IArticle, since this is an interface for one article ?